waku/v2/node/config.nim

```diff
@@ -76,10 +77,14 @@ type
       name: "rln-relay" }: bool
     
     rlnRelayMemIndex* {.
-      desc: "(experimental) the index of node in the rln-relay group: a value between 0-49 inclusive",
-      defaultValue: 0
-      name: "rln-relay-membership-index" }: uint32
-    
+      desc: "(experimental) the index of node in the rln-relay group: a value between 0-99 inclusive",
+      defaultValue: MembeshipIndex(0)
+      name: "rln-relay-membership-index" }: MembeshipIndex
+
+    rlnRelayPubsubTopic* {.
+      desc: "the pubsub topic for which rln-relay gets enabled",
+      defaultValue: ""
```

Comment:
Does it make sense to define a sensible default topic here, such as `waku/2/rlnrelay/proto`? Otherwise, with `rln` defined the node may consider an empty string as valid configuration.

Reply:
Good idea, although an empty topic would not get through (rln relay will not consider it valid) since it is checked against all the pubsub topics supported at the relay level (that there would be no support for an empty topic).

Reply:
done in https://github.com/status-im/nim-waku/pull/722/commits/58589923801d2dbef6463367f76275718f8f5f64